src/mcp-handler.ts

```diff
@@ -49,6 +50,8 @@ async function createMcpServer(options: Options, client: string) {
   const server = new McpServer({
     name: pkgJson.name,
     version: pkgJson.version,
+  }, {
+    instructions: loadPrompt({ promptFileName: 'server.txt' })
```

Comment:
let's just make it an `.md` file instead and import it as a regular import like we do in https://github.com/storybookjs/addon-mcp/blob/main/src/tools/get-ui-building-instructions.ts#L4

Then we don't need the `loadPrompt`-function at all 💪

Reply:
I like this! I'll take a swing at this a bit later when I have some more time. Should be before the next couple of days 😃